### PR TITLE
3.1.x: Release Mender 3.1.0

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Review by tranchitella — APPROVED** [review]

**Review by tranchitella — APPROVED** [review]
The helm chart portion is correct

**Comment by lluiscampos** [comment]
See also https://github.com/mendersoftware/mender-docs-changelog/pull/11

File: 02.Overview/14.Compatibility/docs.md [hunk 1]
```diff
@@ -90,7 +90,8 @@ Clarification of the table:
 | Mender client 2.4.x             | community                 | community             | stable             |
 | Mender client 2.5.x             | community                 | community             | stable             |
 | Mender client 2.6.x             | community                 | community             | stable             |
-| Mender client 3.0.x             | no                        | no                    | stable<sup>3</sup> |
+| Mender client 3.0.x             | no                        | community             | stable<sup>3</sup> |
```
Comment (kacf) [inline 02.Overview/14.Compatibility/docs.md:93]: This should go to the 3.0.x branch as well, I suppose.

**Review by kacf — APPROVED** [review]

File: 02.Overview/14.Compatibility/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 02.Overview/14.Compatibility/docs.md:93]: Thanks, I'll port it.

Comment (lluiscampos) [reply 02.Overview/14.Compatibility/docs.md:93]: @kacf It is already there: https://github.com/mendersoftware/mender-docs/blob/master/02.Overview/14.Compatibility/docs.md#mender-client-and-yocto-project-version

How this was missing from master is a mystery (that I won't solve today)

**Pull request merged**